.github/workflows/ci.yml

```diff
@@ -895,8 +899,10 @@ jobs:
         python-version: ${{ matrix.python }}
 
     - name: Prepare env
+      # Ensure use of NumPy 2 (via NumPy nightlies but can be changed soon)
       run: |
         python3 -m pip install -r tests/requirements.txt
+        python3 -m pip install --upgrade --pre -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy scipy
```

Comment:
Can we have an explicit version check here? To make sure that numpy 2 actually gets installed / got installed?

Reply:
Sure